fcrepo-parent/pom.xml

```diff
@@ -69,6 +69,17 @@
     <sonar.jacoco.itReportPath>${jacoco.outputDir}/${jacoco.out.it.file}</sonar.jacoco.itReportPath>
   </properties>
 
+  <distributionManagement>
```

Comment:
I would suggest using the existing `distributionManagement` block: https://github.com/fcrepo4/fcrepo4/blob/20520a066407778627cf8500b49d039fbd80599e/fcrepo-parent/pom.xml#L571-L586
..and potentially align the `serverId` (https://github.com/fcrepo4/fcrepo4/pull/1673/files#diff-44094b67d49071d47bb32efe43678722R402).